src/Calibration.cpp

```diff
@@ -6,6 +6,8 @@
 #include "DriverLog.h"
 #include "Quaternion.h"
 
+static const char* c_poseSettingsSection = "pose_settings";
```

Comment:
Ideally we'd only define these in one location, now it's in both the `DeviceProvider.cpp` file and here

Reply:
Resolved this in a later commit